pages/_app.tsx

```diff
@@ -2,13 +2,13 @@ import type { AppProps } from "next/app";
 import { ChainId, ThirdwebProvider } from "@thirdweb-dev/react";
 import "../styles/globals.css";
 
-// This is the chainId your dApp will work on.
-const activeChainId = ChainId.Polygon;
+// This is the chain your dApp will work on.
+const activeChain = "mumbai";
```

Comment:
hey, this should be polygon not mumbai

Reply:
hey sorry about these i have made changes to all my open PR that needed a change, i wasnt thinking and started switching unused contract ones like the signin button to mumbai for consistency